graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/DgsQueryExecutor.kt

```diff
@@ -101,6 +101,16 @@ interface DgsQueryExecutor {
      */
     fun <T> executeAndExtractJsonPath(query: String, jsonPath: String, variables: Map<String, Any>): T
 
+    /**
+     * Executes a GraphQL query, parses the returned data, and uses a Json Path to extract specific elements out of the data.
+     * The method is generic, and tries to cast the result into the type you specify. This does NOT work work Lists. Use [executeAndExtractJsonPathAsObject] with a [TypeRef] instead.
```

Comment:
Typo on
>This does NOT work work Lists..

Reply:
Good catch! Inherited typo :)  fix https://github.com/Netflix/dgs-framework/pull/302/commits/994694274bb34cae2fe2d117f41de4117ddb24d2

Reply:
Thanks for fixing it in the other places!